pkgs/applications/networking/irc/shout/default.nix

```diff
@@ -0,0 +1,49 @@
+{ stdenv, fetchFromGitHub, callPackage, python, utillinux }:
+
+with stdenv.lib;
+
+let
+  nodePackages = callPackage (import ../../../../top-level/node-packages.nix) {
+    neededNatives = [ python ] ++ optional (stdenv.isLinux) utillinux;
+    self = nodePackages;
+    generated = ./package.nix;
```

Comment:
@svanderburg:
Serioulsy, there is no better way than including a 5000 lines file for each package?